### ISSUE TITLE
Allow opting out of checking for interfaces

### ISSUE THREAD

**Issue by hmil** [hmil/tslint-override#5]
This has been discussed in https://github.com/Microsoft/TypeScript/issues/2000 , but I will only implement it upon request or if a PR is initiated.

**Comment by stasberkov** [comment]
I was going to submit alike issue but saw it already exist. Not sure my team will like marking every member of interface as "override". Opting out interfaces will be really great.

**Comment by hmil** [comment]
I think this can be implemented by adding the following line [here](https://github.com/hmil/tslint-override/blob/master/rules/explicitOverrideRule.ts#L311) or something similar.

```ts
if (clause.token === ts.SyntaxKind.ImplementsKeyword) continue;
```

Of course this feature needs a config flag (`exclude-implements` or `exclude-interfaces` sounds right) and some tests in `test/test.ts` and `test/spec.ts.lint`.

Feel free to submit a PR.

**Comment by stasberkov** [comment]
will do) Thank you!

**Comment by hmil** [comment]
Available since v0.3.0 behind the `exclude-interfaces` option thanks to @stasberkov 👍
